### PR TITLE
[ADMIN] Add postal code filtering by multiple comma separated value

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Admin/EventAdmin.php [hunk 1]
```diff
@@ -197,8 +197,14 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                         return;
                     }
 
-                    $qb->andWhere(sprintf('%s.postAddress.postalCode', $alias).' LIKE :postalCode');
-                    $qb->setParameter('postalCode', $value['value'].'%');
+                    $value = array_map('trim', explode(',', strtolower($value['value'])));
+                    $idExpression = $qb->expr()->orX();
```
Comment (B-Galati) [inline src/Admin/EventAdmin.php]: Why `idExpression` ?

File: src/Admin/AdherentAdmin.php [hunk 2]
```diff
@@ -300,8 +300,14 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                         return;
                     }
 
-                    $qb->andWhere(sprintf('%s.postAddress.postalCode', $alias).' LIKE :postalCode');
-                    $qb->setParameter('postalCode', $value['value'].'%');
+                    $value = array_map('trim', explode(',', strtolower($value['value'])));
```
Comment (ottaviano) [inline src/Admin/AdherentAdmin.php]: what about move this code into a class (ex `PostalCodeQueryBuilderUtils`) with static method for doing the stuff for avoiding the duplication?
`'callback' => [PostalCodeQueryBuilderUtils::class, 'doStuff']`

Comment (Remg) [reply src/Admin/AdherentAdmin.php]: in this case a custom `AdminExtension` should be appropriate: 
https://symfony.com/doc/master/bundles/SonataAdminBundle/reference/extensions.html

File: src/Admin/EventAdmin.php (hunk 1 above)
Comment (Remg) [reply src/Admin/EventAdmin.php]: like our lord and savior @HeahDude once said: "copy pasted with the ass."

File: src/Admin/Extension/PostalCodeFilterAdminExtension.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    public function configureDatagridFilters(DatagridMapper $datagridMapper)
+    {
+        $datagridMapper
+            ->add('postalCode', CallbackFilter::class, [
+                'label' => 'Code postal (préfixe)',
+                'show_filter' => true,
+                'field_type' => TextType::class,
+                'callback' => function (ProxyQuery $qb, string $alias, string $field, array $value) {
+                    if (!$value['value']) {
+                        return;
```
Comment (Remg) [inline src/Admin/Extension/PostalCodeFilterAdminExtension.php]: `return false;`
This indicates whether the filter is applied on this field or not

**Pull request merged**